proto/minder/v1/minder.proto

```diff
@@ -2143,3 +2143,33 @@ message Provider {
     // This is an output-only field. It may be: "set", "unset", "not_applicable".
     string credentials_state = 9;
 }
+
+// These are for persistence use only, and are not part of the API contract
+
+// GitHubRepoID is the external ID for a repository in GitHub
+message GitHubRepoID {
```

Comment:
I'm not sure I understand this part, is the idea then to serialize this and use it as the external ID? Thought we'd use the GH ID for this as it allows us to detect if there has been a repo change or takeover.